stock_quantity_history_location/static/src/xml/inventory_report.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-extend="StockInventoryReport.Buttons">
+        <t t-jquery="button.o_button_at_date" t-operation="inner">
```

Comment:
We only must change the button description if we have group_stock_multi_locations security group.